neuralmonkey/config/builder.py

```diff
@@ -163,7 +170,9 @@ def instantiate_class(name, all_dicts, existing_objects, depth):
     return obj
 
 
-def build_config(config_dicts, ignore_names, warn_unused=False):
+def build_config(config_dicts: Dict[str, Any],
+                 ignore_names: Set[str],
+                 warn_unused=False) -> Dict[str, Any]:
```

Comment:
těžko odvodíš, že je to bool, když to může bejt cokoliv typu `Union[bool, doplň_svuj_oblíbenej_typ, ...]`